.github/workflows/unit-test-shards.json

```diff
@@ -270,5 +270,11 @@
     "paths": [
       "common/djangoapps/"
     ]
+  },
+  "xmodule-1": {
+    "settings": "lms.envs.test",
+    "paths": [
+      "xmodule/"
+    ]
```

Comment:
You had originally added an `xmodule-2` shard, which tested the folder using CMS settings. I thought that was a great idea. Any specific reason you decided to remove it?

Reply:
Actually when the `xmodule` folder was in `common` the tests were also being executed by just `lms` settings... so I just tried to keep it that way.

Reply:
Fair enough.